pkg/controller/controller_binding.go

```diff
@@ -430,11 +474,11 @@ func (c *controller) reconcileServiceInstanceCredential(binding *v1alpha1.Servic
 	}
 
 	// All updates not having a DeletingTimestamp will have been handled above
```

Comment:
I find this comment to be a bit convoluted and more confusing that it needs to be. Can we instead say something more simpler like

> When we reach this point, we are dealing with either a delete or an orphan mitigation.